extensions/amp-story/0.1/amp-story-consent.css

```diff
@@ -164,7 +164,7 @@
 }
 
 .i-amphtml-story-consent-action-accept {
-  background: #000000 !important;
+  background: var(--primary-color, #000000) !important;
   color: #FFFFFF !important;
```

Comment:
The text color becomes interesting here, since publishers can use any color (including `#FFFFFF`) as their `--primary-color`.  I think we can optimize this by [calculating the contrast ratio](https://ux.stackexchange.com/a/82068) in JavaScript.  We can calculate both:

- The contrast ratio between `--primary-color` and `#FFFFFF`
- The contrast ratio between `--primary-color` and `#000000`

Then use whichever has the greater contrast ratio by either setting the color in the `style` attribute, or using this to apply a CSS class.

Reply:
Thanks for the formula!
If it's fine with you I'd like to work on this on a separate PR (which I'd be doing right now)

Reply:
SGTM